Institue-of-software-technologies/ist-feedback
# Fix/update favicon

### Fix/update favicon

This branch addresses the following:

- Updated the favicon to a new design.
- Ensured compatibility across all supported browsers.
- Verified the favicon displays correctly on both light and dark themes.
- Fixed any issues with incorrect favicon paths in the project.

#### To test:
1. Open the site in a browser.
2. Confirm the updated favicon is displayed correctly.
3. Verify that the favicon works on multiple tabs and devices.

Please review and merge once tested. 👍
